src/renderHook.tsx

```diff
@@ -42,12 +43,20 @@ export function renderHook<Result, Props>(
     return null;
   }
 
-  const { rerender: baseRerender, unmount } = render(
+  const wrap = (element: React.ReactElement) =>
```

Comment:
What is the purpose of this change? It seems to do the same thing as previous code with just passed wrapper to `renderWithAct`

Reply:
`renderWithAct` does what `render` does but without detecting hostComponentNames